docs/get-started-with-datahub.md

```diff
@@ -0,0 +1,62 @@
+# Get Started With DataHub
+
+This section is focused on helping you get up and running with DataHub as quickly as possible.
+
+## Get Started with Self-Hosted DataHub
+
+This is a great place to start if you're new to DataHub and are interested in taking the open-source project for a spin.
+
+You'll learn how to:
+
+* Deploy DataHub on your local machine
+* Ingest sample data to get a feel for what is possible in DataHub
+* Add and invite users to DataHub
+* Troubleshoot common issues
+
+### Prerequisites
+
+Before you go further, ensure you have the following installed:
+
+* [Docker](https://docs.docker.com/get-docker/)
```

Comment:
We also need Python right?

Reply:
Ah good point - what version do we require?

Reply:
Looks good!